Octokit.Tests.Integration/Clients/RepositoryContentsClientTests.cs

```diff
@@ -128,5 +128,53 @@ public class TheGetContentsMethod
                 Helper.DeleteRepo(repository);
             }
         }
+
+        [IntegrationTest]
+        public async Task GetsArchiveLinkAsTarball()
+        {
+            var github = new GitHubClient(new ProductHeaderValue("OctokitTests"))
```

Comment:
That's what `ctrl-c, ctrl-v` gets you, shouldn't trust previous methods in same class :innocent: (_just kidding_)